src/styles/colors.ts

```diff
@@ -199,6 +199,7 @@ const getColorsByTheme = (darkMode?: boolean) => {
     [ChainId.avalanche]: '#E84142',
     [ChainId.degen]: '#A36EFD',
     [ChainId.blast]: '#25292E',
+    [ChainId.apechain]: '#0054FA',
```

Comment:
ok i missed this one on my PRs, is it possible to use native assets color? or maybe we're missing this color from the backend data

Reply:
Looked into this, they're different from native asset colors so I dunno. @christianbaroni any idea on this?